src/test/java/io/supertokens/test/passwordless/api/PasswordlessUserGetAPITest2_11.java

```diff
@@ -159,8 +159,8 @@ public void testGoodInput() throws Exception {
 
         PasswordlessStorage storage = StorageLayer.getPasswordlessStorage(process.getProcess());
 
-        String userIdEmail = "userId";
-        String userIdPhone = "userIdPhone";
+        String userIdEmail = "userIdSampleRandomWithLengththiemail";
```

Comment:
This could be a valid UUID as well, since the length requirement isn't really apparent and it seems so random. I'm fine with this as well, I just prefer if these kind of requirements are explicit or if it's valid data.

Reply:
noted.

Reply:
I saw that you replaced this with random data in base64, but this isn't really what I meant as that's not a valid UUID.
This works but it's not really what I meant:
1. Just keep "userIdWhatever..." and add a comment about why it needs to be a specific length.
2. Generate a new UUID like some tests do elsewhere (`io.supertokens.utils.Utils.getUUID();`)
3. Hardcode a valid UUID.

Reply:
Makes sense, proceeded with the 1st recommendation.